workout-history.php

```diff
@@ -0,0 +1,26 @@
+<?php
+    include('include/init.php');
+
+
+    $workoutHistory = GetWorkouts($_SESSION['userId']);
+
+
+    foreach($workoutHistory as $workout){
+        echo "<b>".$workout['workoutName']."</b><br>";
+        echo $workout['dateTimeEnded']."<br>";
+
+        $workoutSets = GetSetsAndMovementsForWorkout($workout['workoutId']);
+        //debugOutput($workoutSets);
+        $currentMovementName = NULL;
+
+        foreach($workoutSets as $sets){
+            if($currentMovementName == NULL or $currentMovementName != $sets['movementName']){
```

Comment:
Is there a reason why you chose to use the keyword 'or' rather than the operator `||`? If there is no reason for this choice I think you might be better off with the more commonly used `||` so avoide unexpected behavior like the right-hand side of your expression getting ignored due to precedence. See (https://www.php.net/manual/en/language.operators.logical.php) and (https://www.php.net/manual/en/language.operators.precedence.php#:~:text=The%20expression%201%20%3C%3D%201%20%3D%3D%201%20on%20the%20other%20hand%20is%20legal%2C%20because%20the%20%3D%3D%20operator%20has%20a%20lower%20precedence%20than%20the%20%3C%3D%20operator). I think the second link with the example is helpful for wrapping your head around this